packages/ui/src/index.tsx

```diff
@@ -6,7 +6,6 @@ export { default as Text } from './text'
 export { default as Input } from './input'
 export { default as Button } from './button'
 export { default as LoadingDots } from './loading-dots'
-export { default as A } from './anchor'
```

Comment:
Are this barrels correctly tree-shaken?

Reply:
They should be, yes, the package esports ESModules and Webpack should be able to tree shake it without issues